app/connectors/CustomsDataStoreConnector.scala

```diff
@@ -60,4 +61,24 @@ class CustomsDataStoreConnector @Inject()(http: HttpClientV2,
     }
   }
 
+  def verifiedEmail(implicit hc: HeaderCarrier): Future[EmailVerifiedResponse] = {
+    val emailDisplayApiUrl = s"${appConfig.customsDataStore}/subscriptions/email-display"
+
+    httpClient.get(url"$emailDisplayApiUrl").execute[EmailVerifiedResponse].recover {
```

Comment:
We have traditionally broken these into their own lines for each. Like this:

get(xyz)
.execute[123]
.recover {
abc
}

Reply:
Done